src/Matcher/Pattern/Expander/InArray.php

```diff
@@ -51,4 +51,12 @@ public function getError()
     {
         return $this->error;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'inArray';
```

Comment:
Could you please make constant from those names? For example:

`InArray::NAME `

This should allow to use constants here https://github.com/coduo/php-matcher/blob/master/src/Parser/ExpanderInitializer.php#L18

You can also replace `"Coduo\\PHPMatcher\\Matcher\\Pattern\\Expander\\StartsWith"` with `StartsWith::class`